db/migrate/20181203003146_create_users.rb

```diff
@@ -0,0 +1,17 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string   :name
+      t.string   :email
+      t.string   :encrypted_password
+      t.string   :battle_tag
```

Comment:
might want a ToonHandle field in addition to the IDs listed here. this is the unique identifier used to track players in replays (invariant through btag changes, but somewhat difficult to acquire reliably without uploading a replay). We don't have to use it right away, but in the future if someone's btag changes, you'll have a hard time looking them up in the match database if their toonhandle is unknown

Reply:
Yeah, that's a good one.